distribution/docs/src/main/jdocs/content/_security-framework/realms-contents.adoc

```diff
@@ -0,0 +1,107 @@
+:title: Realms
+:type: subSecurityFramework
+:status: published
+:parent: Subject
+:order: 01
+:summary: Shiro realms.
+
+${branding} uses Realms for Authentication and Authorization.
+
+===== Authenticating Realms
```

Comment:
❓ What is a `Realm`?

Reply:
Took a stab at adding a definition and a link to Shiro docs.